packages/preview/curryst/0.1.0/README.md

```diff
@@ -0,0 +1,71 @@
+# Prooftree
```

Comment:
The name isn't correct

Reply:
Thank you ! This has been changed is the last commit.